GitUI/CommandsDialogs/SettingsDialog/CheckSettingsLogic.cs

```diff
@@ -41,7 +41,7 @@ private bool SolveEditor()
 
             if (string.IsNullOrEmpty(editor))
             {
-                GlobalConfigFileSettings.SetPathValue("core.editor", EditorHelper.FileEditorCommand);
+                Environment.SetEnvironmentVariable(CommonLogic.GitEditorEnv, EditorHelper.FileEditorCommand);
```

Comment:
Is this the only value set automatically?
If "check at startup" is not set, is the environment value set? (i.e. when is the backup env var set?)

Reply:
> If "check at startup" is not set, is the environment value set? (i.e. when is the backup env var set?)

I missed that. I am adding the missing call of `SolveEditor` (later today).

> Is this the only value set automatically?

Yes, the others are internal:

```
            bool valid = SolveGitCommand();
            valid = SolveLinuxToolsDir() && valid;
            valid = SolveGitExtensionsDir() && valid;
            valid = SolveEditor() && valid;
```

> Only if portable. If not portable it should be in global.

It is just another two clicks when you configure the merge tool (see screenshot).